### PR TITLE
Added Markdown Previewer

### PR DESCRIPTION
**Pull request by AmrutaJayanti**
### Issue Title: Project: Markdown Previewer

- **Name of the project** : Markdown Previewer
- **Category:**  Vanilla
- **Name:**  Amruta Jayanti
- **GitHub ID:** [AmrutaJayanti](https://github.com/AmrutaJayanti)
- **Idenitfy yourself:**  GSSOC Contributor

Closes: #218 

### Describe the add-ons or changes you've made 📃

I added a Markdown Previewer directory in `Vanilla-Js/Advanced`

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Screenshots 📷
Screenshot 2024-05-30 193250

### Note to reviewers 📄

Please approve

### CODE REVIEW

File: Vanilla-JS-Projects/Advanced/Markdown-Previewer/Screenshot.webp
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Advanced/Markdown-Previewer/Screenshot.webp:1]: file name should be `screenshot.webp`

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@AmrutaJayanti don't forget to update your readme.
- But this markdown previewer is able to preview the code part? 
- Work on it to make it a complete application.

**Comment by AmrutaJayanti** [comment]
@Avdhesh-Varshney I made the changes . Please approve

File: Vanilla-JS-Projects/Advanced/Markdown-Previewer/Readme.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+</p>
+
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+
+![MarkdownPreviewer](https://raw.githubusercontent.com/AmrutaJayanti/WebMasterLog/MarkDown/Vanilla-JS-Projects/Advanced/Markdown-Previewer/Screenshot.webp)
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Advanced/Markdown-Previewer/Readme.md:58]: update the link, ss is not visible.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by AmrutaJayanti** [comment]
@Avdhesh-Varshney Sorry for the inconvenience. Please approve

**Review by Avdhesh-Varshney — APPROVED** [review]
@AmrutaJayanti PR Approved 🎉

**Pull request merged**